### PR TITLE
Config subsystem docstrings (D/DAR) 

### PR DESCRIPTION
**Pull request by thedoubl3j**
Removing .flake8 entries for src/ansible_navigator/configuration_subsystem/* and resolving flake8 docstring errors.

### CODE REVIEW

File: src/ansible_navigator/configuration_subsystem/parser.py [hunk 1]
```diff
@@ -30,7 +29,11 @@ def __init__(self, config: ApplicationConfiguration):
 
     @staticmethod
     def generate_argument(entry) -> Tuple[Any, Any, Dict[str, Any]]:
-        """Generate an argparse argument"""
+        """Generate an argparse argument.
+
+        :param entry: Single settings entry
+        :returns: Argparse object
```
Comment (alisonlhart) [inline src/ansible_navigator/configuration_subsystem/parser.py:35]:
```suggestion
        :returns: Long and short cli parameters, and dictionary of parsed arguments 
```

File: src/ansible_navigator/configuration_subsystem/definitions.py [hunk 2]
```diff
@@ -146,7 +149,11 @@ def environment_variable(self, prefix: str = "") -> str:
 
     @property
     def invalid_choice(self) -> str:
-        """Generate an invalid choice message for this entry"""
+        """Generate an invalid choice message for this entry.
+
+        :returns: Constructed message
+        :raises ValueError: If source is not set for that entry
```
Comment (alisonlhart) [inline src/ansible_navigator/configuration_subsystem/definitions.py:155]:
```suggestion
        :raises ValueError: If source is not set for that entry
        :returns: Constructed message
```

Comment (alisonlhart) [reply src/ansible_navigator/configuration_subsystem/definitions.py:155]: Just flipping the two statements so returns is on the end.

File: src/ansible_navigator/configuration_subsystem/definitions.py [hunk 3]
```diff
@@ -161,16 +168,18 @@ def invalid_choice(self) -> str:
...
-        """Generate a dashed version of the name"""
+        """Generate a dashed version of the name.
+
+        :returns: Dashed name
+        """
         return self.name.replace("_", "-")
 
     def settings_file_path(self, prefix: str) -> str:
-        """Generate an effective settings file path for this entry"""
-        if prefix:
```
Comment (cidrblock) [inline src/ansible_navigator/configuration_subsystem/definitions.py:169]: oops :)  These got dropped

Comment (cidrblock) [reply src/ansible_navigator/configuration_subsystem/definitions.py:169]: I think it why the tests are failing

**Comment by thedoubl3j** [comment]
oof let me close this and cherry pick these changes.

**Pull request closed without merging**